specs/marketplace.md

```diff
@@ -34,73 +45,35 @@ The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL N
 | Smart Contract             | A smart contract implementing the marketplace functionality.                                                               |
 | Token               | ERC20-based token used within the Codex network.     |
 
-## Motivation
-
-The Codex network aims to create a peer-to-peer storage engine with robust data durability, data persistence guarantees, and a comprehensive incentive structure.
-
-The marketplace is a critical component of the Codex network, serving as a platform where all involved parties interact to ensure data persistence. It provides mechanisms to enforce agreements and facilitate data repair when SPs fail to fulfill their duties.
-
-Implemented as a smart contract on an EVM-compatible blockchain, the marketplace enables various scenarios where nodes assume one or more roles to maintain a reliable persistence layer for users. This specification details these interactions.
+## Semantics 
 
-The marketplace contract manages storage requests, maintains the state of allocated storage slots, and orchestrates SP rewards, collaterals, and storage proofs.
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”,
+“SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-A node that wishes to participate in the Codex persistence layer MUST implement one or more roles described in this document.
+The marketplace contract manages storage requests,
+maintains the state of allocated storage slots, and orchestrates SP rewards,
+collaterals, and storage proofs.
+A node that wishes to participate in the Codex persistence layer MUST implement one or
+more roles described in this document.
```

Comment:
Movtivation section does not belong in the semantic or specification section because it is does not have details about the protocol. More like an introduction to the topic.